apps/api/src/hyp3_api/handlers.py

```diff
@@ -80,13 +79,12 @@ def get_names_for_user(user):
 
 
 def get_user(user):
-    max_jobs, _, remaining_jobs = dynamo.jobs.get_quota_status(user)
+    user_record = dynamo.user.get_user(user)
```

Comment:
```suggestion
    # TODO: maybe create a helper function for this since we do the same thing in `put_jobs`
    user_record = dynamo.user.get_user(user)
```